pkg/users/local.js

```diff
@@ -498,8 +498,22 @@ PageAccountsCreate.prototype = {
 
     create: function() {
         var tasks = [
-            function create_user() {
-                var prog = ["/usr/sbin/useradd", "--create-home", "-s", "/bin/bash"];
+            () => {
+                const dfd = cockpit.defer();
```

Comment:
Weird, normal JS Promises should chain just fine..

Reply:
https://github.com/cockpit-project/cockpit/blob/master/pkg/users/local.js#L188
We use `done` and `fail`.